content/guides/exporters/supported-exporters/Go/Honeycomb.md

```diff
@@ -43,8 +41,15 @@ func main() {
     exporter := honeycomb.NewExporter("YOUR-HONEYCOMB-WRITE-KEY", "YOUR-DATASET-NAME")
     defer exporter.Close()
 
-    trace.RegisterExporter(exporter)
+    sampleFraction := 0.5
+    trace.ApplyConfig(trace.Config{DefaultSampler: trace.ProbabilitySampler(sampleFraction)})
+    // If you use the Open Census Probability Sampler, be sure to pass that sampleFraction to the exporter
+    // so that Honeycomb can pick it up and make sure we handle your sampling properly.
+    // Note: The Probability Sampler uses a fraction, whereas Honeycomb uses an integer, which is the inverse of that fraction.
```

Comment:
nit: this explainer might not be necessary anymore, since the exporter accepts the `sampleFraction` as-is; at this point, a user doesn't need to care that libhoney uses integer sample rates internally.